commit 41e2a941ad0108858475882f545b0f9adbe7b7b2 (refs/changes/55/3055/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-12-11T22:09:35+00:00 (1 year, 10 months ago)
    
    OS-6499 dlled uses wrong link name when the link doesn't exist
    Reviewed by: Ryan Zezeski <rpz@joyent.com>
    Approved by: Ryan Zezeski <rpz@joyent.com>

diff --git a/usr/src/cmd/dlutil/dlled.c b/usr/src/cmd/dlutil/dlled.c
index 65f2cdc6f7..f6d6a7aab2 100644
--- a/usr/src/cmd/dlutil/dlled.c
+++ b/usr/src/cmd/dlutil/dlled.c
@@ -268,7 +268,7 @@ main(int argc, char *argv[])
 			if ((status = dladm_name2info(dlled_hdl, argv[i],
 			    &linkid, NULL, NULL, NULL)) != DLADM_STATUS_OK) {
 				(void) fprintf(stderr, "failed to get link "
-				    "id for link %s: %s\n", link,
+				    "id for link %s: %s\n", argv[i],
 				    dladm_status2str(status, dlled_dlerrmsg));
 				return (1);
 			}
